src/common/core/users/serialization.js

```diff
@@ -1031,6 +1061,8 @@ define(['common/util/assert', 'blob/BlobConfig'], function (ASSERT, BlobConfig)
                     core.setPointer(node, keys[i], null);
                 } else if (nodes[target] && toRemoveGuids.indexOf(target) === -1) {
                     core.setPointer(node, keys[i], nodes[target]);
+                } else if (isCombinedId(target)) {
+                    core.setPointer(node, keys[i], getCombinedTarget(target));
                 } else {
                     console.log('error handling needed???!!!???');
```

Comment:
How about throwing an exception?
It was probably meant not to be entered before either but it did.
Now if someone imports and doesn't pay attention to the logs they might end up with an incomplete model